RecoTracker/MeasurementDet/plugins/MeasurementTrackerEventProducer.cc

```diff
@@ -49,6 +60,9 @@ MeasurementTrackerEventProducer::produce(edm::Event &iEvent, const edm::EventSet
     edm::ESHandle<MeasurementTracker> measurementTracker;
     iSetup.get<CkfComponentsRecord>().get(measurementTrackerLabel_, measurementTracker);
 
+    edm::ESHandle<SiPixelFedCablingMap> cablingMap;
+    iSetup.get<SiPixelFedCablingMapRcd>().get(pixelCablingMapLabel_, cablingMap);
```

Comment:
to change it to conditional fetch, there should also be an interface change in the ```::updatePixels``` method
to avoid dereferencing an empty handle.